src/utils/frontEndApiClient/scheduleOfRates/codes.test.js

```diff
@@ -22,6 +22,30 @@ describe('getSorCodes`', () => {
         contractorReference: 'H01',
         propertyReference: '00001234',
         tradeCode: 'PL',
+        isRaisable: false,
```

Comment:
This can be deleted as the `isRaisable` is an optional param and we will only provide it when is true